### DISCUSSION TITLE
feeding options to dcm2niix when called by dcm2bids_helper and dcm2bids

### DISCUSSION THREAD

**Discussion by rcwelsh** [UNFmontreal/Dcm2Bids#190] [Q&A]
Due to some automated series generation on Siemens when running a multi-echo sequence and also when distortion correction is turned on, the default options inside dcm2bids/utils.py result in not all of the images being properly reconstructued. There does not seem to be an option for either dcm2bids_helper or dcm2bids to override the default options of :

`dcm2niixOptions = "-b y -ba y -z y -f '%3s_%f_%p_%t'"`

This would be a good thing to add to the package -- unless someone has the CLI solution.

**Comment by arnaudbore** [comment]
Hi @rcwelsh ,

You just need to add your specific parameters to the config file directly.
Hope it helps.
Next time send your questions on Neurostars with dcm2bids tag. 
We don't really look into the github discussions.

Best,
Arnaud
